### ISSUE TITLE
Make grid units configurable (GUI scaling for users)

### ISSUE THREAD

**Issue by Flohack74** [ubports/ubuntu-touch#226]
Currently all scaling configuration lives in /etc/ubuntu-touch-session.d/ and is checked on boot for overrides. It would be rather easy, let´s say trivial, to inject a custom hook into /usr/bin/ubuntu-touch-session which is a small script, to make an additional check for /home/phablet/.session.config and if it exists to include the gu ratio from there.

Thereby users can determine how big or small the overall GUI is scaled. later on we can put this to system-settings ofc.

**Comment by dpniel** [comment]
If a user doesn't like the defaults then they can already just create an upstart job with no other modifications needed. 

Example:

`/home/phablet/.config/upstart/gu.conf`
```
start on starting dbus
exec initctl set-env --global GRID_UNIT_PX=15
```

System settings could just create/overwrite the upstart job if you user wants to adjust the default

**Comment by Flohack74** [comment]
Closing since workaround available.

**Comment by dobey** [comment]
I don't think we should be closing bugs or feature requests, simply on the basis of workarounds being possible, @Flohack74 Can we please re-open this?

**Comment by Flohack74** [comment]
I do think we should close tickets like this one, since we are sitting in front of an overwhelming amount of unresolved stuff and we need to focus on more important things. And its my ticket, and I am happy with the workaround. And no other people joined in so far, so it seems to be of less importance.

**Comment by doniks** [comment]
i've added my +1 to the issue to mark my 'join'. in case that sways your opinion

**Comment by UniversalSuperBox** [comment]
While this likely shouldn't be taken on until we upgrade Unity8 and Mir, this issue is still a valid feature request.

**Comment by dobey** [comment]
Now that UTTT has an option to change this setting, do we need to keep this open? I don't think new unity8/mir are related at all though. If anything, I'd say some major changes to the toolkit are necessary to implement this properly. And no matter how we implement it for native things, scaling will likely still be an issue with HTML and apps using other toolkits.
